components/raftstore-v2/src/raft/storage.rs

```diff
@@ -15,7 +15,10 @@ use raft::{
     eraftpb::{ConfState, Entry, Snapshot},
     GetEntriesContext, RaftState, INVALID_ID,
 };
-use raftstore::store::{util, EntryStorage, ReadTask, RAFT_INIT_LOG_INDEX, RAFT_INIT_LOG_TERM};
+use raftstore::store::{
+    util, util::new_empty_snapshot, EntryStorage, ReadTask, WriteTask, RAFT_INIT_LOG_INDEX,
```

Comment:
It only used in test, should be `use` in test part.

Reply:
fixed